src/instructlab/utils.py

```diff
@@ -541,3 +541,20 @@ def split_hostport(hostport: str) -> tuple[str, int]:
     if not hostname or not port:
         raise ValueError(f"Invalid host-port string: '{hostport}'")
     return hostname, port
+
+
+def is_oci_repo(repo_url: str) -> bool:
+    """
+    Checks if a provided repository follows the OCI registry URL syntax
+    """
+
+    # TO DO: flesh this out and make it a more robust check
+    oci_url_prefix = "docker://"
+    return repo_url.startswith(oci_url_prefix)
+
+
+def is_tool_present(name: str) -> bool:
```

Comment:
You don't need a function, you can do this directly in an `if` statement. Thanks

Reply:
is this not something that we may want to reuse in the future potentially?

Reply:
There isn't much to re-use when the function is just line :)

Reply:
removed, thanks!